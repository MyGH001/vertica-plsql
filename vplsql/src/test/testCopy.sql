/**
* testing copy statement
*/

select PLSQL_CREATE(using parameters content=
$$create or replace procedure p_test_copy()
as
begin
  drop table if exists t_test_copy;
  create table t_test_copy(id IDENTITY, txt varchar(256));

  exec 'copy t_test_copy(txt) from ''/etc/hosts'' direct';
  exec 'select count(*)>0 from t_test_copy';
  
  drop table if exists t_test_copy;
end;$$);

select PLSQL_EXEC($$p_test_copy()$$);

select PLSQL_DROP(using parameters name='p_test_copy');
