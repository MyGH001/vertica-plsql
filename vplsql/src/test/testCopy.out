Ln:1 CREATE PROCEDURE p_test_copy
t
t
